packages/fx-core/templates/plugins/resource/apim/bicep/apimConfiguration.bicep

```diff
@@ -0,0 +1,41 @@
+param apimServiceName string
```

Comment:
this seems to be a provision operation, can we put it to apimProvision.bicep?

Reply:
Fixed.